README.md

```diff
@@ -1,436 +1,95 @@
+* visits the Homepage -> call metrilo.viewPage
+* visits an article page -> call metrilo.viewArticle
+* visits a category page -> call metrilo.viewCategory
+* visits a product page -> call metrilo.viewProduct
+* visits another page(e.g. /thank-you) -> call metrilo.viewPage
+* types something in the search bar -> call metrilo.search
+* adds a product to his cart -> call metrilo.addToCart
+* removes a product to his cart -> call metrilo.removeFromCart
+* checks the content of his cart -> call metrilo.viewPage
+* goes to the checkout page -> call metrilo.checkout
+* enters his email in some html form -> call metrilo.identify
 
-The following example shows how identification works:
-```javascript
-var identifyParams = { name: 'Johny Bravo', first_name: 'Johny', last_name: 'Bravo', email: 'johnybravo@gmail.com' };
+### About the identify
+When a customer comes to the store and he is not loged in Metrilo treats him as an anonymous visitor. Metrilo adds a unique identifier to his session, so it could attribute his actions to this identifier. Once he provides his email(e.g. logs in, signs up, fills in a subscription form, makes an order, etc.) you have to call ```metrilo.identify(customerEmail)```. This will associate the unique identifier with the email. Moreover, all events that the visitor has while being an anonymous one will be merged with the events he has as identified(*if any). Keep in mind that the identify call must be done only once, the first time the visitor enters his email. If you are not sure that you have already called it, just check for the existence of email in the cookie named **cbuid**. If there is an email, no need to call it. This cookie contains either the unique identifier or the email and part of the identify function is to change the content  from the unique identifier to the email.
 
-metrilo.identify('johnybravo@gmail.com', identifyParams);
-```
-_First name and last name are not required_
+**\*** sometimes customers might log out, use different devices, browsers or delete their cookies. Every time one of this happens Metrilo will create a new anonymous visitor with a new unique identifier. However, once the visitor identifies himself(enters his email and you call metrilo.identify) Metrilo will check for the existence of a customer with the entered email and if there is any, it will merge him with the anonymous visitor.
 
-In addition to the `identifyParams` you can include:
-* `subscribed` boolean field to let Metrilo know if the customer is subscribed to receive emails
-* `tags` key with array of tags to apply
-* company
-* country
-```javascript
-var identifyParams = {
-  name: 'Johny Bravo',
-  first_name: 'Johny',
-  last_name: 'Bravo',
-  email: 'johnybravo@gmail.com',
-  subscribed: true,
-  tags: ['early_subscriber'],
-  company: 'Metrilo',
-  country: 'US'
-};
-metrilo.identify('johnybravo@gmail.com', identifyParams);
-```
 
-### Tracking events ###
+## Backend integration
```

Comment:
This one should go to a separate file as well (+ code examples).

Reply:
same as above